rpc/server_options.go

```diff
@@ -33,6 +33,9 @@ type serverOptions struct {
 	// allowUnauthenticatedHealthCheck allows the server to have an unauthenticated healthcheck endpoint
 	allowUnauthenticatedHealthCheck bool
 
+	// publicMethods is a list of publicMethods that are allowed to bypass auth for
```

Comment:
```suggestion
	// publicMethods are api routes that attempt, but do not require, authentication
```

[nit] typo in original comment